test/e2e/debug-server.spec.ts

```
@@ -0,0 +1,49 @@
+import { expect, test } from '@playwright/test'
+
+test.describe('Server Debugging', () => {
+  test('should verify server is responding', async ({ page, request }) => {
+    // Test if server root is accessible
+    const rootResponse = await request.get('/')
+    console.log(`Root response status: ${rootResponse.status()}`)
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)